.osparc/osparc-meta-parallelrunner/metadata.yml

```diff
@@ -1,8 +1,8 @@
 name: ParallelRunnerService
 description: "ParallelRunnerService"
 key: simcore/services/dynamic/osparc-meta-parallelrunner
-version: 0.0.9
-integration-version: 0.0.9
+version: 0.0.10
+integration-version: 0.0.10
```

Comment:
Minor, just saw this: normally we don't upgrade the `integration-version` with the service version, I think integration version for dynamic services is always for the moment `2.0.0`

Then I don't remember exactly what `integration version: 2.0.0` means (maybe @GitHK remembers, is it our internal convention to say that those are "new style" dynamic services?)

Reply:
Yes, exactly. I saw this happening as well. 😇

Reply:
If the latter is the case it probably means we need to fix the bump version script (which is in another image), because it would also fail I guess once a one reaches service version 2.0.0

Reply:
The other image that bumps the version, [uses the command `bump2version` installed via pip](https://github.com/ITISFoundation/ci-service-integration-library/blob/main/requirements/_base.in#L9) and the `.bumpversion.cfg` file in this repo (or the one one in the jupyter-math repo). 
 I think we can fine-tune `.bumpversion.cfg` so that it skips the `integration-version` (some more info might be [here](https://github.com/peritus/bumpversion?tab=readme-ov-file#configuration)).